contents/docs/user-guides/recordings.mdx

```diff
@@ -34,10 +33,28 @@ To do so, you should add the CSS class name `ph-no-capture` to elements which sh
 
 Additionally, when dealing with inputs, if you wish to still capture the input box but not its contents, you can use the class name `ph-ignore-input` instead.
 
-## Controlling the volume of recordings
+## Recordings data retention
+
+Depending on how your app/website is built, recordings can take a lot of disk space. To manage this, we have the following retention policy options in place.
+
+### PostHog Cloud & Clickhouse Self-hosted
+
+If you're running a Clickhouse instance, recordings are **automatically deleted after 3 weeks**. Old recordings are deleted with Clickhouse's <a target="_blank" href="https://clickhouse.com/docs/en/sql-reference/statements/alter/ttl/">Table TTL</a>. You can set your own retention policy by executing a query like this on your Clickhouse instance.
```

Comment:
I think we should make it clearer that only self-hosted users can set their own retention policy. What about:

> By default, recordings are **automatically deleted after 3 weeks**. Old recordings are deleted with Clickhouse's <a target="_blank" href="https://clickhouse.com/docs/en/sql-reference/statements/alter/ttl/">Table TTL</a>. If you're self-hosting, you can set your own retention policy by executing a query like this on your Clickhouse instance.

Reply:
```suggestion
### PostHog Cloud & Clickhouse Self-hosted

By default, recordings are automatically deleted after 3 weeks. Old recordings are deleted with Clickhouse's <a target="_blank" href="https://clickhouse.com/docs/en/sql-reference/statements/alter/ttl/">Table TTL</a>. If you're self-hosting, you can set your own retention policy by executing a query like this on your Clickhouse instance.
```